app/models/katello/content_view_version_export_history.rb

```diff
@@ -8,17 +8,18 @@ class ContentViewVersionExportHistory < Katello::Model
 
     belongs_to :content_view_version, :class_name => "Katello::ContentViewVersion", :inverse_of => :export_histories
     validates_lengths_from_database
+
     validates :content_view_version_id, :presence => true
     validates :destination_server, :uniqueness => { :scope => [:content_view_version_id, :destination_server, :path] }
-    validates :export_type, :inclusion => { :in => EXPORT_TYPES,
-                                            :allow_blank => false,
-                                            :message => _("Invalid export_type from one of the following: %s" % EXPORT_TYPES.join(', '))
-                                          }
-
+    validates :export_type, :presence => true,
+              :inclusion => { :in => EXPORT_TYPES,
+                              :allow_blank => false,
+                              :message => _("Invalid export_type from one of the following: %s" % EXPORT_TYPES.join(', '))
```

Comment:
While we're here, this message could be clearer:
```suggestion
                              :message => _("must be one of the following: %s" % EXPORT_TYPES.join(', '))
```